src/xcode/ENA/ENA/Source/Scenes/QRScanner/QRScannerCoordinator.swift

```diff
@@ -95,7 +95,7 @@ class QRScannerCoordinator {
 	private let coronaTestService: CoronaTestService
 	private let recycleBin: RecycleBin
 
-	private let activityIndicatorView = QRScannerActivityIndicatorView()
+	private let activityIndicatorView = QRScannerActivityIndicatorView(frame: .zero, title: AppStrings.FileScanner.hudText)
```

Comment:
Can't we have a separate initializer that calls the other one with `frame: .zero` internally? Seems a bit weird to have to pass in the frame in times of auto layout.